docs/docs/Versioning_And_Upgrade/Backports_And_Hotfixes.md

```diff
@@ -0,0 +1,49 @@
+# Backports and Hotfixes
+
+If your plugin uses an ["enterprise-style"](/Versioning_And_Upgrade/Versioning.md#enterprise-style-release-strategy) release strategy, then you'll probably want to occasionally provide new "minor" or "patch" versions that build atop older versions.
+
+Code changes that are applied atop old releases are usually called "backports". Sometimes, they are also called "hotfixes", if the change is specifically created for a single user.
+
+These releases present a problem: although they are built atop an older code branch, they are still newer than some releases from a newer code branch. Below, we'll walk through how we prevent users from "upgrading" to a new-branch release that would be incompatible with an installed old-branch release.
+
+### Motivating Example
+Let's take a look at an example of a possible timeline of releases.
+
+> **February**: The initial version of a plugin is released, with build number "1.0". This is a simple plugin that uses a simple strategy for syncing dSources.
+
+> **April**: A new version is released, with build number "1.1". This adds some bugfixes and adds some small optimizations to improve the performance of syncing.
+
+> **August**: A new version is released, with build number "2.0". This uses a completely new syncing strategy that is far more sophisticated and efficient.
+
+Let's assume that not all users will want to upgrade to the 2.0 release immediately. So, even months later, you expect to have a significant number of users still on version 1.0 or 1.1.
+
+Later, in October, a bug is found which impacts all releases. This bug is important enough that you want to fix it for **all** of your end users (not just the ones using 2.0).
+
+Here are the behaviors we need:
+
+* Our 2.0 end users should be able to get the new bugfix without giving up any of the major new features that were part of 2.0.
+* Our 1.0 and 1.1 end users should be able to get the new bugfix without also needing to accept all the major new features that were part of 2.0.
+* Once an end user has received the bugfix, it should be impossible to lose the bugfix in an upgrade.
+
+### Strategy
+
+You can include a [data migration](/Versioning_And_Upgrade/Upgrade.md#data-migrations) along with your bugfix. If your bugfix involves a schema change, you will have to do this anyways. If not, you can still include a data migration that simply does nothing. If a user with the bugfix attempts to "upgrade" to 2.0, the Delphix Engine will prevent it, because the 2.0 releases does not include this migration.
```

Comment:
The reason is there, its to prevent an unexpected upgrade:
> If a user with the bugfix attempts to "upgrade" to 2.0, the Delphix Engine will prevent it, because the 2.0 releases does not include this migration.